cmd/cloud_login.go

```diff
@@ -0,0 +1,178 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"syscall"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+	"gopkg.in/guregu/null.v3"
+
+	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/consts"
+	"go.k6.io/k6/ui"
+)
+
+const cloudLoginCommandName = "login"
+
+type cmdCloudLogin struct {
+	globalState *state.GlobalState
+}
+
+func getCmdCloudLogin(gs *state.GlobalState) *cobra.Command {
+	c := &cmdCloudLogin{
+		globalState: gs,
+	}
+
+	// loginCloudCommand represents the 'cloud login' command
+	exampleText := getExampleText(gs, `
+  # Log in with an email/password.
+  {{.}} cloud login
+
+  # Store a token in k6's persistent configuration.
+  {{.}} cloud login -t <YOUR_TOKEN>
+
+  # Display the stored token.
+  {{.}} cloud login -s
+  
+  # Reset the stored token.
+  {{.}} cloud login -r`[1:])
```

Comment:
Nit; but I think we don't use dots at the end of the example titles for the rest of the commands.

```suggestion
  # Log in with an email/password
  {{.}} cloud login

  # Store a token in k6's persistent configuration
  {{.}} cloud login -t <YOUR_TOKEN>

  # Display the stored token
  {{.}} cloud login -s
  
  # Reset the stored token
  {{.}} cloud login -r`[1:])
```